force-app/main/default/lwc/graphqlPagination/__tests__/graphqlPagination.test.js

```diff
@@ -0,0 +1,96 @@
+import { createElement } from 'lwc';
+import GraphqlPagination from 'c/graphqlPagination';
+import { graphql } from 'lightning/uiGraphQLApi';
+
+// Mock realistic data
+const mockGraphQL = require('./data/graphqlPaginationResponse.json');
+
+describe('c-graphql-pagination', () => {
```

Comment:
@mburr-salesforce shouldn't we add a test that covers navigation to the next page and check the following:
1. we use `endCursor` correctly in the next query
2. the page counter increments

Reply:
I added a new test that advances to the next page & added checks the states of both buttons & the status text. LMK if there are additional things you'd like to have the test check.